Source/UnitEx.cpp

```diff
@@ -960,6 +960,23 @@ void UnitEx::SetLaunchPadCargo(map_id moduleType)
 	(*unitArray)[unitID].launchPadCargo = moduleType;
 }
 
+map_id UnitEx::GetConVecCargoTurret()
+{
+	if (!isInited) {
+		return static_cast<map_id>(HFLNOTINITED);
+	}
+
+	if (!IsLive() || (GetType() != mapConVec)) {
+		return static_cast<map_id>(-1);
+	}
+
+	if (GetCargo() == mapGuardPost) {
+		return static_cast<map_id>((*unitArray)[unitID].conVecCargoTurret);

```

Comment:
I think we want to start migrating to static_casts where appropriate.

Reply:
Agreed, `static_cast` should be preferred over C-style casts. Though no casts would be best, where that is achievable.